freshmaker/config.py

```diff
@@ -463,6 +484,10 @@ def set_item(self, key, value):
         # Create the empty self._key attribute, so we can assign to it.
         setattr(self, "_" + key, None)
 
+        self.create_property(key)
+        self.create_attribute(key, value)
```

Comment:
Sorry, I didn't get, what was the point to add this?

Reply:
I split the code to sub methods for better decomposition @mulaievaRH

Reply:
IMO, I think it would be better to not split it, because `create_property` and `create_attribute` should always be called together and maybe also in order, when you split it to two functions, there is no mechanism to make sure they will be called in the right way (this will bring some confusions to future devs).